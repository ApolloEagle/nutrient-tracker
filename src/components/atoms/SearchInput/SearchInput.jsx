import React from 'react';
import { TextField, InputAdornment } from '@mui/material';

const SearchInput = ({ icon, ...otherProps }) => (
  <TextField
    {...otherProps}
    InputProps={{
      startAdornment: <InputAdornment position="start">{icon}</InputAdornment>,
    }}
  />
);

export default SearchInput;
