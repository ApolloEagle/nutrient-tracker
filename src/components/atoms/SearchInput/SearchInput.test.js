import React from 'react'
import { shallow } from 'enzyme';
import SearchInput from "./SearchInput";
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const wrapper = shallow(<SearchInput />);

describe("Search Input", () => {
  it("Renders search input", () => {
    expect(wrapper).toMatchSnapshot();
  });
});