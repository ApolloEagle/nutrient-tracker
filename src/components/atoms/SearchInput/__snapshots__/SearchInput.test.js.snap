// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Input Renders search input 1`] = `
<ForwardRef(TextField)
  InputProps={
    Object {
      "startAdornment": <ForwardRef(InputAdornment)
        position="start"
      />,
    }
  }
/>
`;
